azkaban-web-server/src/web/js/azkaban/view/flow-execute-dialog.js

```diff
@@ -496,18 +498,14 @@ var disableFinishedJobs = function(data) {
       node.status = "READY";
       node.disabled = true;
     }
-    else if (node.status == "SUCCEEDED" || node.status=="RUNNING") {
+    else if (node.status == "SUCCEEDED" || node.missing) {
       node.disabled = true;
     }
-    else if (node.status == "CANCELLED") {
```

Comment:
Thanks a lot for education. Actually I thought **cancelled** is equal to FAILED_FINISHING.